src/app/page.tsx

```diff
@@ -0,0 +1,51 @@
+import styles from "./page.module.css";
+import blurBackgroundSvg from "../../public/svg/blur-background.svg";
+import donggleLogoSvg from "../../public/svg/donggle-logo.svg";
+import donggleExamplePng from "../../public/image/donggle-example-png-2x.png";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function Introduce() {
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <div className={styles.logo}>
+          <Image src={donggleLogoSvg} width={20} height={20} alt='동글 로고' />
```

Comment:
```suggestion
          <Image src='/svg/donggle-logo.svg' width={20} height={20} alt='동글 로고' />
```

public 폴더에 이렇게 접근할 수 있어서 바로 경로를 작성해도 좋아보여요.!

Reply:
오...! 이렇게 동작할 수 있다는거 이제 알았네요! ㅎㅎ👍